docs/section22/Rule22-17.md

```diff
@@ -0,0 +1,52 @@
+
+# CHW&CW - Rule 22-17  
+
+**Schema Version:** 0.0.25    
+**Mandatory Rule:** True  
+**Rule ID:** 22-17  
+**Rule Description:** The baseline heat rejection device shall have an efficiency of 38.2 gpm/hp  
```

Comment:
We are not checking that this rating is at conditions specified in Table 6.8.1-7. I think this is OK, but perhaps worth including in a note accompanying the outcome.

Reply:
I don't think that we have the data to check whether this is at the specified rating conditions.  I added a note to the PASS result